src/app/components/Header/index.jsx

```diff
@@ -6,13 +6,20 @@ import {
   GEL_SPACING_DBL,
   GEL_SPACING_HLF,
 } from '../../lib/constants/styles';
+import { layoutGridWrapper, layoutGridMain } from '../../lib/layoutGrid';
 
 const StyledHeader = styled.header`
+  ${layoutGridWrapper}
   background-color: ${C_POSTBOX};
   height: 80px;
   width: 100%;
   padding: ${GEL_SPACING_DBL}px;
 `;
+
+const GridDiv = styled.div`
```

Comment:
Also it would be good to rename `GridDiv` to  `GridItem` or if you want to definitely have the tag name, `GridItemDiv`.